python/batch-predictor/requirements.txt

```diff
@@ -2,5 +2,5 @@ pyspark==2.4.5
 mlflow==1.6.0
 cloudpickle==2.0.0
 pyarrow==0.14.1
-protobuf>=3.0
+protobuf>=3.0,<4.0.0dev
```

Comment:
How about using `<4`?

Reply:
I am following this https://github.com/googleapis/python-bigquery/blob/51c1f3713ebad7d186a46ffe6b5580a33158655d/setup.py#L45. I presume <4 will also include pre-release package.